README.md

```diff
@@ -1,67 +1,5 @@
-![FTL-SDK](https://dl.dropboxusercontent.com/u/20701844/tachyon.png) FTL-SDK
+This project has moved to https://dev.azure.com/mixer-oss/Mixer/_git/ftl-sdk
 
-FTL-SDK is a cross platform SDK written in C to enable sending audio/video to mixer using our FTL service
+If you have the appropriate persmissions on this repo, please make it read-only to prevent further changes.
```

Comment:
Please remove those lines from the readme. Then I'll merge and r/o the repo

Reply:
Also typo.

Reply:
Thanks @Doridian !

@QuinnDamerell and @sslivins are asking that we don't take this down from GitHub, so let's keep this PR blocked for now, until/unless we get their approval.